lib/data_migrate/data_migrator.rb

```diff
@@ -38,7 +38,7 @@ def schema_migrations_table_name
       end
 
       def migrations_path
-        "db/data"
+        DataMigrate.config.data_migrations_path || "db/data"
```

Comment:
I see `"db/data"`, i wonder if it's possible to set the default where this is set

Reply:
What do you mean?
I think the clearest solution is to set `config.data_migrations_path = "db/data", so that would be the default value. When you want to override it in your application, you could specify it using initializers

Reply:
yes, that's what I'm suggesting. @EnomaDebby what do you think?

Reply:
I think that works